setup.cfg

```
@@ -296,7 +297,7 @@ console_scripts =
     onegov-gazette = onegov.gazette.cli:cli
     onegov-landsgemeinde = onegov.org.cli:cli
     onegov-org = onegov.org.cli:cli
-    onegov-people = onegov.people.cli:cli
+    onegov-parliament = onegov.parliament.cli:cli
```

```suggestion
    onegov-people = onegov.people.cli:cli
```
parliament doesn't have any CLI commands (yet), and you removed the people specific CLI commands.